.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Docker Image
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and push
```

Comment:
Given we're not pushing the image can we give this step a clearer name

Reply:
The copy/paste gremlins have been crushing me lately :laughing: